sspl_test/run_qa_test.sh

```diff
@@ -313,8 +337,9 @@ fi
 
 if [ "$IS_VIRTUAL" == "true" ]
 then
-    echo "Stoping the SSPL service"
-    $sudo systemctl stop sspl-ll
+    # Start mock API server if virtual machine
+    echo "Starting mock server on 127.0.0.1:$MOCK_SERVER_PORT"
+    $script_dir/mock_server &
```

Comment:
why delaying mock server start ?

Reply:
I just noticed some scenarios when we are activating the mock server before SSPL restart. SSPL will raise an alert for mocked data before the SSPL getting restart for the SSPL test.
So I have made some sequence changes that 
1) Stop SSPL first 
2) Start the mock server
3) Start SSPL
So I feel this is the correct way to absorb mock data only for the SSPL tests.

Reply:
right, agree